app/controllers/responsiblepeople/YourResponsiblePeopleController.scala

```diff
@@ -21,24 +21,29 @@ import connectors.DataCacheConnector
 import controllers.BaseController
 import models.responsiblepeople.ResponsiblePerson
 import utils.RepeatingSection
-import views.html.responsiblepeople.your_answers
+import views.html.responsiblepeople.your_responsible_people
 
-trait YourAnswersController extends RepeatingSection with BaseController {
+trait YourResponsiblePeopleController extends RepeatingSection with BaseController {
 
-  val dataCacheConnector: DataCacheConnector
+  def dataCacheConnector: DataCacheConnector
 
   def get() =
       Authorised.async {
         implicit authContext => implicit request =>
           dataCacheConnector.fetch[Seq[ResponsiblePerson]](ResponsiblePerson.key) map {
-            case Some(data) => Ok(your_answers(data))
+            case Some(data) => {
+              val s = ResponsiblePerson.filterWithIndex(data)
```

Comment:
val (completeRP, incompleteRP) = ...
and pass into view.

your_responsible_people(completeRP, incompleteRP)

Reply:
done